org.freefilesync.FreeFileSync.json

```diff
@@ -0,0 +1,108 @@
+{
+  "app-id": "org.freefilesync.FreeFileSync",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "FreeFileSync",
+  "finish-args": [
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host"
+  ],
+  "build-options": {
+    "cflags": "-O2 -g",
+    "cxxflags": "-O2 -g"
+  },
+  "cleanup": [
+    "/include",
+    "/lib",
+    "/share/doc",
+
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "boost",
+      "buildsystem": "simple",
+      "no-make-install": true,
+      "build-commands": [
+        "./bootstrap.sh --prefix=/app",
+        "./b2 install -j `nproc` --with-chrono --with-system --with-thread variant=release link=static threading=multi runtime-link=static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.65.1/source/boost_1_65_1.tar.gz",
+          "sha256": "a13de2c8fbad635e6ba9c8f8714a0e6b4264b60a29b964b940a22554705b6b60"
+        }
+      ]
+    },
+    {
+      "name": "wxWidgets",
```

Comment:
I'll wait until after the build finishes to be sure but I believe wxwidgets needs an rm-configure and then a reconfigure with the local toolchain like so: https://github.com/flathub/com.bixense.PasswordCalculator/blob/d08810ce993f1052e9ec78bc1841989e930f6ddf/com.bixense.PasswordCalculator.json#L14 otherwise you'll fail with aarch64

Reply:
Yeah, as suspected it failed: https://flathub.org/builds/#builders/6/builds/1044

Reply:
I added `rm-configure: true`. Not sure if that was it or something else is needed. Let's try the build and see.

Reply:
you're going to need to something like:

```
{
                    "type": "script",
                    "dest-filename": "autogen.sh",
                    "commands": [
                        "autoreconf -vfi"
                    ]
                }
```

To the sources are as well.

Reply:
That doesn't work:
```
aclocal: warning: autoconf input should be named 'configure.ac', not 'configure.in'
configure.in:58: error: possibly undefined macro: AC_BAKEFILE_PROG_CC
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf: /usr/bin/autoconf failed with exit status: 1
Error: module wxWidgets: module wxWidgets: Child process exited with code 1
```

I guess I'll need your help here, because I have no experience with autotools (or compiling C programs for that matter), so I don't really understand what we're trying to achieve here, and can't even test it on aarch64, just amd64.

Reply:
Sure, I'll take a specific look when I have a moment.